src/lang/en.json

```diff
@@ -624,6 +624,7 @@
     "matrixDesc1": "You can find the internal room ID by looking in the advanced section of the room settings in your Matrix client. It should look like !QMdRCpUIfLwsfjxye6:home.server.",
     "matrixDesc2": "It is highly recommended you create a new user and do not use your own Matrix user's access token as it will allow full access to your account and all the rooms you joined. Instead, create a new user and only invite it to the room that you want to receive the notification in. You can get the access token by running {0}",
     "Channel Name": "Channel Name",
+    "Mention Channel": "Mention Channel",
```

Comment:
> Just to reiterate - it's presence makes Slack notify the user, the exact notification can depend on user preferences, but by default its an audible alert. Otherwise the message only shows up as an unread message and not something that should necessarily be attended to i.e., it would be used for notifications in your company's #Ops channel to alert a group of people to an immediate issue. It's use by a non-bot, regular user would be equivalent to `@channel`

I my concern is, that this behaviour would not be obvious to end users.
Do you think a more descriptive label would be appropriate? (would this label be obvious to slack users?)

Reply:
@CommanderStorm would definitely be obvious to me and the teams I work with. Maybe "Notify Channel" is better? I normally see the `@` referred to as a `mention` regardless of the channel/team/user it precedes.

Reply:
Yes, that would probably be a better wording.

I think a description would be appropriate nevertheless (silent vs nonsilent mentiones are a nonstandard feature and new users might not be aware of this).

Reply:
@CommanderStorm That change, adding the description is up. Screenshot attached:
![screenshot_20230601_084218](https://github.com/louislam/uptime-kuma/assets/715994/becc0217-eb97-46e8-98f0-cbc5bd89fa5e)